EC-Switch-Seclog.py

```diff
@@ -0,0 +1,973 @@
+    print('   -o --org             : The AWS profile of the Organisation account')
+    print('   -v --verbose         : Debug mode - optional')
+
+
+def get_account_id(session):
+    """
+    This function gets te id of the account defined in the profile
+        :param force: flag to force the retrieval of the account ID 
+        :return: a string with the account id 
+    """
+
+    try:
+        sts = session.client('sts')
+        response = sts.get_caller_identity()
+        return response['Account']
+    except ClientError as error:
+        if error.response['Error']['Code'] == 'AccessDenied':
+            print(f"Access denied getting account id [{Status.FAIL.value}]")
+            print("Exiting...") 
+            sys.exit(1)
+        else:
+            raise error
+
+def is_seclog(session):
+    """
+    Function that checks if the account is a seclog account
+        :return: true or false
+    """
+    client = session.client('ssm')
+    seclog_account_id = get_account_id(session)
+    response = client.get_parameter(Name='/org/member/SecLogMasterAccountId')
+    if not 'Value' not in response or seclog_account_id != response['Parameter']['Value']:
+        return False
+    return True
+
+def remove_stacks_from_stackset(stackset,account_id, session):
+    
+    global regions
+
+    print(f"Remove stacks from StackSet {stackset} in progress ", end="")
+
+    cfn = session.client('cloudformation')
+    response = cfn.describe_stack_set(StackSetName=stackset)
+   
+    if response['StackSet']['Status'] not in ('ACTIVE'):
+        print(f"Cannot remove stacks from stackset {stackset}. Current stackset status is: {response['StackSet']['Status']} [{Status.FAIL.value}]")
+        print("Exiting...")
+        sys.exit(1)
+   
+    with Spinner():
+        filter=[{
+            'Name': 'DETAILED_STATUS',
+            'Values': 'PENDING'
+        }]
+        response = cfn.list_stack_instances(StackSetName=stackset,Filters=filter)
+
+        while(len(response['Summaries']) > 0):
+            time.sleep(1)
+            response = cfn.list_stack_instances(StackSetName=stackset,Filters=filter)
+
+        try:
+            operationPreferences={
+                'RegionConcurrencyType': 'PARALLEL',
+                'FailureToleranceCount': 9,
+                'MaxConcurrentCount': 10
+            }
+            cfn.delete_stack_instances(
+                StackSetName=stackset, 
+                Regions=regions, 
+                Accounts=[account_id],
+                OperationPreferences=operationPreferences,
+                RetainStacks=False
+                )
+
+            time.sleep(5)
+            response = cfn.list_stack_set_operations(StackSetName=stackset)
+            while(any(x['Status'] == "RUNNING" for x in response['Summaries'])):
+                time.sleep(10)
+                response = cfn.list_stack_set_operations(StackSetName=stackset)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `list_stack_set_operations` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)